test/groups.yaml

```diff
@@ -87,6 +86,20 @@ localvolumeprovisioner:
     - "defaultstorageclass-protection"
     - "localvolumeprovisioner"
 
+# ------------------------------------------------------------------------------
+# Dispatch
+#
+# All Dispatch related addons should be tested as a part of this group
+# ------------------------------------------------------------------------------
+dispatch:
+    - "cert-manager"
+    - "defaultstorageclass-protection"
+    - "localvolumeprovisioner"
```

Comment:
if you are having trouble getting volumes provisioned, use the default storage path, and remove these: `"defaultstorageclass-protection" and `"localvolumeprovisioner"`